lib/gltfPipeline.js

```diff
@@ -53,17 +53,7 @@ function processJSONWithExtras(gltfWithExtras, options, callback) {
     convertDagToTree(gltfWithExtras);
     combineMeshes(gltfWithExtras);
     combinePrimitives(gltfWithExtras);
-
-    // run the AO stage
-    if (options.ao_diffuse) {
-        var aoOptions = {
-            scene: options.ao_scene,
-            rayDepth: options.ao_rayDepth,
-            numberSamples: options.ao_samples,
-            resolution: options.ao_resolution
-        };
-        bakeAmbientOcclusion(gltfWithExtras, aoOptions);
-    }
+    bakeAmbientOcclusion(gltfWithExtras, options.aoOptions);
```

Comment:
Oh that makes sense... no idea why someone would have aoOptions but then not run the AO.